driver/driver_test.go

```diff
@@ -861,3 +763,33 @@ func createContrailNetwork(c *controller.Controller) *types.VirtualNetwork {
 	return controller.CreateMockedNetworkWithSubnet(
 		c.ApiClient, networkName, subnetCIDR, project)
 }
+
+func deleteHNSEndpoint(d *ContrailDriver) {
+	hnsNets, err := d.hnsMgr.ListNetworks()
+	Expect(err).ToNot(HaveOccurred())
+	eps, err := hns.ListHNSEndpointsOfNetwork(hnsNets[0].Id)
+	Expect(err).ToNot(HaveOccurred())
+	hnsEndpointID := eps[0].Id
+	err = hns.DeleteHNSEndpoint(hnsEndpointID)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func getTheOnlyHNSEndpoint(d *ContrailDriver) (*hcsshim.HNSEndpoint, string) {
```

Comment:
I think deleteHNSEndpoint and getTheOnlyHNSEndpoint should both follow the same name convention, ex. deleteOneHNSEndpoint and getOneHNSEndpoint